github-runner-manager/src/github_runner_manager/cli.py

```diff
@@ -61,17 +56,17 @@ def main(  # pylint: disable=too-many-arguments, too-many-positional-arguments
 
     Args:
         config_file: The configuration file.
-        openstack_config_file: The OpenStack configuration file.
-        host: The hostname to listen on for the HTTP server.
+        host: The hostname to listen on for the HTTP server
         port: The port to listen on the HTTP server.
         debug: Whether to start the application in debug mode.
     """
     if debug:
         logging.basicConfig(level=logging.DEBUG)
 
     lock = Lock()
-    config = ApplicationConfiguration.from_yaml_file(config_file)
-    openstack_config = OpenStackConfiguration.from_yaml_file(openstack_config_file)
+    config_str = config_file.read()
+    config = ApplicationConfiguration.from_yaml_file(StringIO(config_str))
+    openstack_config = OpenStackConfiguration.from_yaml_file(StringIO(config_str))
```

Comment:
This reads a bit confusing to me. In my mind, when I read `from_yaml_file`, I would expect the yaml file to be equal to the config, and not just contain parts of it. Does this make sense?

I would add the openstack config as subobject to the ApplicationConfiguration.

Reply:
This is why originally the there was two configuration files.

During the refactor last month, the ApplicationConfiguration and OpenStackConfiguration created, but not merged. I will ask Javi on why, and see if they can be merged.

Reply:
Updated.